sdk/keyvault/Azure.ResourceManager.KeyVault/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.4.0 (2025-02-24)
```

Comment:
if you would like to release a stable version, you will have to change the version number in csproj file as well to be align with the version number here.